modules/serverless-quarkus-functions-overriding-liveness-readiness.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies
+//
+// * functions/reference/serverless-developing-quarkus-functions.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-quarkus-functions-overriding-liveness-readiness_{context}"]
+= Overriding liveness and readiness probe values
+
+You can override `liveness` and `readiness` probe values for your Quarkus functions. This allows you to configure health checks performed on the function.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving are installed on the cluster.
```

Comment:
```suggestion
* The {ServerlessOperatorName} and Knative Service are installed on the cluster.
```
See my earlier question to check if you really meant `Serving` instead of `Service`.

Reply:
Again, it's `Serving` in this case.